src/app/downloads/downloads-chart.component.ts

```diff
@@ -122,11 +122,9 @@ export class DownloadsChartComponent implements OnDestroy {
         case INTERVAL_MONTHLY:
           return monthYearFormat;
         case INTERVAL_WEEKLY:
-          return dayMonthDateFormat;
         case INTERVAL_DAILY:
-          return dailyDateFormat;
+          return dayMonthDateFormat;
```

Comment:
The chart used to have the day of the week. This was based on user input that day of the week is useful info and that charts should start on either Sundays or Mondays rather than be rolling 7 days or whatever That same user contradicted himself in the last testing session and said that he wants rolling 7 day windows. Further, having the day of the week degrades somewhat for charts with more than 9 data points. It will only show the date on every other data point, so the start of the week is not marked, which I think is probably kinda what the user was getting at for wanting to know the day fo the week, so it didn't seem that useful. We might get some negative feedback on this set of changes because of removing the time pickers, so I figured it was as good a time as any to test out these proposed date and time formats.